prices.html

```diff
@@ -305,21 +309,37 @@
     </div>
 
     <div class="accs-price-table-wrapper">
-        <table id="table-network-traffic" class="accs-price-table">
+        <table id="table-loadbalancing" class="accs-price-table">
```

Comment:
I don't think LB should be a separate table. I would have added it to the Network Services table.

Reply:
Updated as suggested.